crates/astria-cli/src/types.rs

```diff
@@ -73,11 +73,11 @@ impl RollupDeploymentConfig {
     }
 
     #[must_use]
-    pub fn get_initial_sequencer_height(&self) -> u64 {
-        self.sequencer.initial_block_height
+    pub fn get_initial_sequencer_height(&self) -> String {
+        self.sequencer.initial_block_height.to_string()
     }
 
-    pub fn set_initial_sequencer_height(&mut self, new_height: u64) {
+    pub fn set_initial_sequencer_height(&mut self, new_height: String) {
```

Comment:
You could make this `new_height: impl Display` and then `self.sequencer.initial_block_height = new_height.to_string();`